git-session.html

```diff
@@ -90,6 +90,14 @@
                <li>Facebok</li>
             </ul>
          </div>
+         <div class="mySlides fade">
+          <h1>Feature six </h1>
+          <ul>
+             <li>Blog</li>
+             <li>Forum</li>
```

Comment:
Hi @Syed-Umair 
Both List is having same name, can check this.

Reply:
sure